flutter_local_notifications_web/web/notifications_service_worker.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+// Experimental function to save a notification to the IndexedDB database
```

Comment:
As it's not used, can this be removed? This to expose on what is necessary. Besides that, I assume if this was left used by others, it creates additional maintenance overhead in tracking what could be a breaking change if removed